lib/travis/build/script/dart.rb

```diff
@@ -236,6 +236,10 @@ def dartfmt
             end
 
             sh.cmd 'unformatted=`dartfmt -n .`'
```

Comment:
Let's not worry about that for now. I just want to land this so folks stop getting a false positive from `dartfmt`